cluster/gce/gci/configure.sh

```diff
@@ -245,7 +245,7 @@ function install-node-problem-detector {
       local -r npd_version="${DEFAULT_NPD_VERSION}"
       local -r npd_hash="${DEFAULT_NPD_HASH}"
   fi
-  local -r npd_tar="node-problem-detector-${npd_version}.tar.gz"
+  local -r npd_tar="node-problem-detector-${npd_version}-linux_amd64.tar.gz"
```

Comment:
Please don't hard-code. Use the `HOST_PLATFORM` and `HOST_ARCH` variables, e.g., https://github.com/kubernetes/kubernetes/blob/15d3c3a5e2679493d5cbe9b944df4ee717057a06/cluster/gce/gci/configure.sh#L303

Thanks!

Reply:
Fixed. Thanks.